hickle/hickle_bshuf.py

```diff
@@ -0,0 +1,49 @@
+# encoding: utf-8
+"""
+Plugin BitShuffleLz4
+
+Based on hickle.dump and hdf5plugin
+
+Hdf5plugin background information: https://github.com/kiyo-masui/bitshuffle
+"""
+
+import hickle
+
+
+class BitShuffleLz4:
+
+    
+    def __init__(self):
+        try:
+            import hdf5plugin
+        except:
```

Comment:
`             except:  # pragma: no cover`

Reply:
There was no point in catching an exception (overkill) as the message in the traceback is clear enough.